.github/workflows/test.yml

```diff
@@ -126,7 +126,7 @@ jobs:
 
   UniTests:
     name: 'U: C++ Unit Tests'
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
     steps:
 
     - uses: actions/setup-python@v4
```

Comment:
Please check the python version available on ubuntu 22.04. https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

Now the min. version is 3.10+
Please update and check